test/src/examples_complete_test.go

```diff
@@ -89,22 +98,22 @@ func TestExamplesComplete(t *testing.T) {
 	// Run `terraform output` to get the value of an output variable
 	eksClusterId := terraform.Output(t, terraformOptions, "eks_cluster_id")
 	// Verify we're getting back the outputs we expect
-	assert.Equal(t, "eg-test-eks-cluster", eksClusterId)
+	assert.Equal(t, "eg-test-eks-"+attributes[0]+"-cluster", eksClusterId)
```

Comment:
attribute := attributes[0]

```suggestion
	assert.Equal(t, "eg-test-eks-"+attribute+"-cluster", eksClusterId)
```